### DISCUSSION TITLE
Serving many small static sites from the same server

### DISCUSSION THREAD

**Discussion by janosrusiczki** [basecamp/kamal-proxy#66] [Q&A]
On my cheap little cloud instance I have quite a few low traffic vhosts. One is a RoR app which would be nice to be deployed via Kamal, there is also a blog built with Jekyll which could also be deployed with Kamal after it is built but the rest are 100% static. One of these static vhosts has all kinds of photos and assets for the blog and the total file size is considerable but the rest are small HTML sites consisting of a few pages and images. Is there a way to serve these through kamal-proxy without the overhead of starting an nginx (or other "passthru" server) Docker image for each?

**Comment by frenkel** [comment]
No, you'll need a simple static file serving container for this.
